docs/.vuepress/config.js

```diff
@@ -46,6 +47,8 @@ module.exports = {
               '/components/card',
               '/components/empty-state',
               '/components/input',
+              '/components/input-checkbox',
```

Comment:
sidebar is not alphabetical now since the title changed

Reply:
Do you think it should stay alpha or should we break it up by type? Im cool reordering
![image](https://user-images.githubusercontent.com/5941111/68787438-656da680-05f6-11ea-8d03-4002755c73a3.png)